routing/router.go

```diff
@@ -1596,21 +1597,29 @@ func (r *ChannelRouter) SendPayment(payment *LightningPayment) ([32]byte, *route
 	return r.sendPayment(payment, paySession)
 }
 
-// SendToRoute attempts to send a payment as described within the passed
-// LightningPayment through the provided routes. This function is blocking
-// and will return either: when the payment is successful, or all routes
-// have been attempted and resulted in a failed payment. If the payment
-// succeeds, then a non-nil Route will be returned which describes the
-// path the successful payment traversed within the network to reach the
-// destination. Additionally, the payment preimage will also be returned.
-func (r *ChannelRouter) SendToRoute(routes []*route.Route,
-	payment *LightningPayment) ([32]byte, *route.Route, error) {
-
-	paySession := r.missionControl.NewPaymentSessionFromRoutes(
-		routes,
+// SendToRoute attempts to send a payment with the given hash through the
+// provided route. This function is blocking and will return the obtained
+// preimage if the payment is successful or the full error in case of a failure.
+func (r *ChannelRouter) SendToRoute(hash lntypes.Hash, route *route.Route) (
+	lntypes.Preimage, error) {
+
+	log.Tracef("Attempting to send payment %x, using route: %v",
+		hash, newLogClosure(func() string {
+			return spew.Sdump(route)
+		}),
 	)
 
-	return r.sendPayment(payment, paySession)
+	preimage, err := r.sendToSwitch(route, hash)
```

Comment:
So we'll now bypass mission control? I think we'd still want to have this information incorporated into any future payment attempts (sendroute or regular).

Reply:
I was already reconsidering this when working on the new mission control. Will revert that.

Reply:
Reverted